docs/udf.rst

```diff
@@ -402,6 +402,34 @@ An example code can be found `here <https://github.com/Open-EO/openeo-python-cli
 
 
 
+Logging from the UDFs
+==============================
+
+In some cases, users might want to use print statements within a UDF so that they can monitor their 
+results that makes debugging easier. One possible way is to use ``inspect`` logging function in the UDF.
```

Comment:
'inspect' is the only way: You can use the inspect logging function to achieve this.
Maybe also turn inspect into a link to the relevant api docs?

Reply:
updated